index.html

```diff
@@ -0,0 +1,275 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Israa Odeh">
+    <meta name="description"
+        content="Welcome to my personal portfolio website! Here, you'll find a comprehensive overview of my education, skills, and professional experience. Explore the various sections to learn more about my background and get in touch.">
+    <title>Israa Odeh | Frontend Developer</title>
+    <link rel="icon" href="images/logo.ico" type="image/x-icon">
+
+    <!--Google Fonts-->
+    <!--Copse Google Font-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Copse&display=swap" rel="stylesheet">
+
+    <!--Delius Google Font-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Copse&family=Delius&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="css/style.css">
+    <script>
+        document.addEventListener('DOMContentLoaded', () => {
+            const menuToggle = document.querySelector('.menu-toggle');
+            const navDropdown = document.querySelector('.nav-dropdown');
+
+            menuToggle.addEventListener('click', () => {
+                navDropdown.classList.toggle('show');
+                const isHidden = navDropdown.getAttribute('aria-hidden') === 'true';
+                navDropdown.setAttribute('aria-hidden', !isHidden);
+                menuToggle.setAttribute('aria-expanded', isHidden);
+            });
+        });
+    </script>
+</head>
+
+<body>
+    <header>
+        <nav class="top-nav"
+            aria-label="The main navigation menu of the portfolio which includes links to the About Me, Skills, Education, Projects, and Contact sections, as well as the logo."
+            tabindex="0">
+            <div class="logo">
```

Comment:
I can't find logo class name in the styles file

Reply:
At first, I was thinking of using it to style the logo, but then I downloaded the already styled logo from my Figma design and forgot to delete this class.

Reply:
Resolved.